### DISCUSSION TITLE
Usd Camera in Stage from File... wire aperture to Resolution Gate from Viewport

### DISCUSSION THREAD

**Discussion by nebukadhezer** [Autodesk/maya-usd#3613] [Q&A] [answered]
Hey,
I was wondering if this is already working and I am doing sth wrong, but I wanted to use a usdcamera in a usd file as the camera for the viewport and it seems like activating the Film- or Resolution Gate does not work...
Should it ?
Or is this not supported atm ? If not it would be a nice addition.
Cheers

**Comment by nebukadhezer** [comment]
Anybody ?

**Comment by wallworm** [comment]
Thanks for your question. I don't have the answer but will bring it up internally.

**Comment by wallworm** [comment, marked as answer]
So we talked about this. It just isn't supported yet. We do want to do some work on improving camera support all around, so we will include this in that work.

**Reply by nebukadhezer** [reply]
Nice thank you for the answer. Is there a rough eta when you might be able to look into this ? Like 3 months or 6 months?

**Reply by wallworm** [reply]
Unfortunately, I am not able to give dates and future timelines. But I can tell you that improving camera workflows is a priority of mine and I intend to have the team do some work in this area.

**Reply by nebukadhezer** [reply]
Thanks for putting prio on it. Looking forward to the updates.

**Reply by nebukadhezer** [reply]
Any news here?

**Comment by santosd** [comment]
@wallworm would you like me to log a ticket for this internally?

**Reply by wallworm** [reply]
Yes, indeed David!

**Reply by nebukadhezer** [reply]
Any news here ?
